types/ejs/ejs-tests.ts

```diff
@@ -2,7 +2,7 @@
 
 import ejs = require("ejs");
 import { readFileSync as read } from 'fs';
-import LRU = require("lru-cache");
+import * as LRU from "lru-cache";
```

Comment:
Is `LRU` used?  If not, would `import "lru-cache";` be clearer?

Reply:
`LRU` is used on line 70: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/af4a799f9e5ce44c4b9eccf65d54562485095c9f/types/ejs/ejs-tests.ts#L70

Reply:
Oops, I missed that.  If it's being invoked, that probably needs to be `import LRU = require("lru-cache");`.

Reply:
done